data/ansible/domain.yml

```
@@ -0,0 +1,14 @@
+version: "3.1"
+
+intents:
+  - intent_ansible_namespaces_access_issues
+
+entities: []
+
+actions: []
+
+slots: {}
+
+responses:
+  utter_inform_reach_out_to_support:
```

```suggestion
  utter_ansible_inform_reach_out_to_support:
```

Just want to scope these. I don't know we would manage to duplicate the name elsewhere, but you never know